master/networking/mtu.md

```diff
@@ -1,129 +1,136 @@
+| Network MTU            | {{site.prodname}} MTU | {{site.prodname}} MTU with IP-in-IP (IPv4) | {{site.prodname}} MTU with VXLAN (IPv4) |
+| ---------------------- | --------------------- | ------------------------------------------ | --------------------------------------- |
+| 1500                   | 1500                  | 1480                                       | 1450                                    |
+| 9000                   | 9000                  | 8980                                       | 8950                                    |
+| 1460 (GCE)             | 1460                  | 1440                                       | 1410                                    |
+| 9001 (AWS Jumbo)       | 9001                  | 8981                                       | 8951                                    |
+| 1450 (OpenStack VXLAN) | 1450                  | 1430                                       | 1400                                    |
 
-Passing in the environment variable `FELIX_IPINIPMTU` (or `FELIX_VXLANMTU`) when running the
-`{{site.nodecontainer}}` container will set the MTU for Felix to use.
+##### Recommended MTU for overlay networking
 
-When using the Kubernetes self-hosted manifests, Felix derives the IP-in-IP value from
-the `veth_mtu` field of the calico-config ConfigMap, which is set to `1440` by default.
+The extra overlay header used in IP in IP and VXLAN protocols, reduces the minimum MTU by the size of the header. (IP in IP uses a 20-byte header, and VXLAN uses a 50-byte header). Therefore, we recommend the following:
 
-### Setting tunnel MTU with calicoctl
+- If you use VXLAN anywhere in your pod network, configure MTU size as “physical network MTU size minus 50”. 
+- If you use only IP in IP, configure MTU size as “physical network MTU size minus 20”
+- Set the workload endpoint MTU and the tunnel MTUs to the same value (so all paths have the same MTU)
 
-To set the IP-in-IP MTU value for all {{site.prodname}} nodes in your cluster, use the
-following command to retrieve the current Felix settings.
+##### MTU for flannel networking
 
-```bash
-calicoctl get felixconfig default --export -o yaml > felix.yaml
-```
+When using flannel for networking, the MTU for network interfaces should match the MTU of the flannel interface. If using flannel with VXLAN, use the “Calico MTU with VXLAN” column in the table above for common sizes. 
 
-Modify ipipMTU (or vxlanMTU) to the intended integer value.
+#### Configure MTU for workloads
 
-```bash
-vim felix.yaml
-```
+When you set the MTU it applies to new workloads. To apply MTU changes to existing workloads, you must restart calico nodes. Restarting the calico/node pods also updates any {{site.prodname}} tunnel network interfaces on that node. 
 
-Replace the current felixconfig settings.
+**Configure MTU in calico-config**
+  
+Edit the `calico-config.yaml` file and update the veth value for your environment. For example:
```

Comment:
What is calico-config.yaml referring to? This isn't something we ship to users today, if I understand correctly.  So I don't think this will make sense to users. 

We might want to say "Edit the ConfigMap in calico.yaml" - but even that is Kubernetes specific and (with the in progress operator work) going away in a future release.

Reply:
@matthewdupre

Reply:
I updated the doc based on our conversation.

Reply:
Fixed.